xarray/tests/test_cftime_offsets.py

```diff
@@ -380,6 +405,43 @@ def test_mul(offset, expected):
     assert offset * 3 == expected
 
 
+@pytest.mark.parametrize(
+    ("offset", "expected"),
+    [
+        (Day(), Hour(n=12)),
+        (Hour(), Minute(n=30)),
+        (Minute(), Second(n=30)),
+        (Second(), Millisecond(n=500)),
+        (Millisecond(), Microsecond(n=500)),
```

Comment:
These are all good tests.  Perhaps one more I would like to see added would be one where more than one iteration through `_next_higher_resolution` is required, e.g. `1e-6 * Second()`.